app/src/ui/generic-git-auth/generic-git-auth.tsx

```diff
@@ -74,6 +75,18 @@ export class GenericGitAuthentication extends React.Component<
               onValueChanged={this.onPasswordChange}
             />
           </Row>
+
+          <Row>
+            <div>
+              Depending on the hosting service of your repository, you might
+              need to use a Personal Access Token (PAT) instead of a password.
+              Take a look at the{' '}
+              <LinkButton uri="https://github.com/desktop/desktop/tree/development/docs/integrations">
+                integration instructions
+              </LinkButton>{' '}
+              for different hosting services to learn how to create a PAT.
```

Comment:
Wondering if we can me more concise.
<LinkButton ...> Learn more about creating a PATs in our integration docs. </LinkButton>